osidb/tests/factories.py

```diff
@@ -495,7 +495,6 @@ class Meta:
     name = factory.sequence(lambda n: f"ps_update_stream_{n}")
     version = factory.Faker("word")
     target_release = factory.Faker("word")
-    rhsa_sla_applicable = factory.Faker("boolean")
```

Comment:
Why is this being removed?

Reply:
Faker will produce random boolean values, but for most tests where `rhsa_sla_applicable` is not specified, we want to use the default which is `True`. Otherwise it will get `rhsa_sla_applicable = False` for some tests and with the change introduced in this commit, the SLA policy will not be generated and the test will fail.

Reply:
I did not realize that there is the default. Makes sense!